lib/failover_ip.rb

```diff
@@ -33,9 +33,12 @@ def current_target
 
     raise unless response.success?
 
-    response.parsed_response["failover"]["active_server_ip"]
+    active_server_ip = response.parsed_response["failover"]["active_server_ip"]
+    $logger.info "Ip of active server: #{active_server_ip}"
```

Comment:
why do you think this line is neccessary? will be quite verbose in the log, as used in multiple locations when a failover occurs - and we already log the current target on failover.

Reply:
the idea is two fold:

1. I assume failover won't and shouldn't be a frequent occurence - as of the previous few times a failover was triggered I've observed it to take about 90s, which I deem an unpleasantly long time - that's why I judge failovers as not occuring regularily
2. eacht time you ping, you get an entry - which makes for a lot of pings and entries. So *one* additional entry should the - rare? - failover really happen "macht den Braten nicht feiss".
3. but the real motivation is: I regard a failover as something uncalled for: if it happens it's rather unpleasant. I definitively want to know why it happened. Also I might be under stress (system is flapping). So I wan't to know at once: where did the system switch from? Where did the system switch to? When did that happen? How long did it take? So me as a sysadmin I don't really want to go hunting backwards through logfiles for the previously active server IP - maybe only to find that that information has long been rotated out, or maybe not logged at all.

Reply:
ok, basically fine with it, but then maybe let's make it even more clear why this log lines appears multiple times by stating that we refetched the current server ip:

```
    $logger.info "Refetched ip of active server: #{active_server_ip}"
```

Reply:
thx -> cecb222 (I've removed the "Re" from "Refetched" since it's not necessarily the case that heardbeat *had* indeed fetched the ip before)